docs/bitcoin/quicksync.md

```diff
@@ -0,0 +1,26 @@
+# QuickSync
+
+<!-- - What is it? -->
+<!-- - How is it useful? -->
+<!-- - What are the tradeoffs? -->
+QuickSync downloads a torrent for a major chunk of the blockchain from MyNode peers without verifying every block.
```

Comment:
It saves time by avoiding the initial blockchain download and this time benefit is significant when using an HDD but marginal with a SSD.
QuickSync is turned on by default and must be disabled from the settings page while setting up to trigger a fully-verified blockchain download.
Once the blockchain is synced with the network, enabling QuickSync will download the torrent and start seeding it.
This helps myNode peers in downloading the blockchain faster during setup. The upload and download speed can be adjusted to optimize bandwidth usage, as shown in the screenshot below.

Reply:
done